modules/cnv-about-datavolumes.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_users_guide/cnv-importing-virtual-machine-images-datavolumes.adoc
+// * cnv/cnv_users_guide/cnv-expanding-virtual-storage-with-blank-disk-images.adoc
+
+[id="cnv-about-datavolumes_{context}"]
+= About DataVolumes
+
+`DataVolume` objects are custom resources provided by the Containerized Data
+Importer (CDI) project. DataVolumes orchestrate import, clone, and upload
+operations associated with an underlying PersistentVolumeClaim (PVC).
+DataVolumes are integrated with KubeVirt, and they can prevent a virtual machine
```

Comment:
"can prevent?" If they always do this, ~can~, otherwise, please explain when they do this.